hyperspy/drawing/utils.py

```diff
@@ -580,6 +582,52 @@ def plot_images(images,
               if sig.axes_manager.navigation_size > 0
               else 1)
 
+    # If no cmap given, get default colormap from pyplot:
+    if cmap is None:
+        cmap = [plt.get_cmap().name]
+    elif cmap == 'mpl_colors':
+        for n_color, c in enumerate(mpl.rcParams['axes.prop_cycle']):
+            make_cmap(colors=['#000000', c['color']],
+                      name='mpl{}'.format(n_color))
+        cmap = ['mpl{}'.format(i) for i in
+                range(len(mpl.rcParams['axes.prop_cycle']))]
+        colorbar = __check_single_colorbar(colorbar)
```

Comment:
`__check_single_colorbar(colorbar)` return `None` in any case and it then removes the colorbar from the plot, by setting `colorbar=None`!!! I would just do the check and not the assign the result of the function to `colorbar`.